actions/lib/action.py

```diff
@@ -10,4 +19,10 @@ def __init__(self, config):
 
     def do_action(self, cls, action, **kwargs):
         obj = getattr(digitalocean, cls)(token=self._token)
-        return getattr(obj, action)(**kwargs)
+        digitalocean_results = getattr(obj, action)(**kwargs)
+        # The results received from the digitalocean module are
+        # custom python objects that StackStorm doesn't recognize.
+        # The following code converts them into dicts & lists
+        json_str = json.dumps(digitalocean_results, default=digitalocean_obj_to_dict)
```

Comment:
Nothing simple I can think of.

This should work for now, as long as the class doesn't contain attributes which are not JSON serializable.

Otherwise the best way would probably be to manually build up the dict and exclude those attributes (we already do this in some other places like aws pack, etc.).